.github/workflows/validate.yml

```diff
@@ -0,0 +1,23 @@
+name: Scheduled validator
+
+on:
+  push:
+
+jobs:
+  validate:
+    name: OPTiMaDe validator
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install OPTiMaDe
+      run: |
+        python -m pip install --upgrade pip
+        pip install git+https://github.com/Materials-Consortia/optimade-python-tools
```

Comment:
I think this should be fixed at a specific version, right? Or do we always want the latest, even though we might make backward incompatible changes?

Reply:
you could simply clone it down, checkout the tagged version of choice and then install.

Reply:
Or just install from pypi ;)

Reply:
> We need to release v0.3.3 first though.

This is now done, so should we update the workflow?

Reply:
Agreed!